crux-test/test/crux/logging_test.clj

```diff
@@ -0,0 +1,58 @@
+(ns crux.logging-test
+  (:require [clojure.test :as t]
+            [clojure.java.io :as io]
+            [clojure.tools.logging :as log]
+            [crux.api :as api]
+            [crux.fixtures :as f]
+            [crux.fixtures.api :as apif :refer [*api*]]
+            [crux.fixtures.kv :as kvf]
+            [crux.fixtures.standalone :as fs]))
+
+(defn remove-log-file [f]
+  (do
+    (io/delete-file "logtester.log")
+    (f)))
+
+(t/use-fixtures :once remove-log-file kvf/with-kv-dir fs/with-standalone-node apif/with-node)
+
+(defn- sync-submit-tx [node tx-ops]
+  (let [submitted-tx (api/submit-tx node tx-ops)]
+    (api/sync node (:crux.tx/tx-time submitted-tx) nil)
+    submitted-tx))
+
+(t/deftest test-submit-tx-log
+  (let [secret 33489857205]
+
+    ;; Put
```

Comment:
you can wrap these in a `(t/testing "put" ...)` block - means that test failures (if/when they happen) contain the extra context information